freqtrade/plugins/pairlist/rangestabilityfilter.py

```diff
@@ -104,6 +106,14 @@ def _validate_pair_loc(self, pair: str, daily_candles: Optional[DataFrame]) -> b
                               f"which is below the threshold of {self._min_rate_of_change}.",
                               logger.info)
                 result = False
+            if pct_change <= self._max_rate_of_change:
```

Comment:
as mentioned above - when using a default of None - you'll need to modify this to first (!) check if max_rate_of_change is None.

Reply:
For clarification (cause I'm a complete amateur lol), by "(!)" do you mean the code would read more like `if self.max_rate_of_change != None`...?

Reply:
Reading your comment again, perhaps you were instead emphasizing the check should be done first?

Reply:
I think you'll need to wrap this whole section into a `if self._max_rate_of_change:`.

you cannot `=` (or `!=`) compare `None` - you'd have to use `x is None` or `x is not None` - but it'll evaluate to falsy without this explicit check - so simply checking this as if it where a boolean is usually sufficient - unless 0.0 is a valid value (which it isn't in this case- as that would mean a pair with 0 movement).

Reply:
> I think you'll need to wrap this whole section into a `if self._max_rate_of_change:`.
Agreed.
 
> you cannot `=` (or `!=`) compare `None` - you'd have to use `x is None` or `x is not None` - but it'll evaluate to falsy without this explicit check - so simply checking this as if it where a boolean is usually sufficient - unless 0.0 is a valid value (which it isn't in this case- as that would mean a pair with 0 movement).
Lol, duh... I actually knew this at some point, and have clearly forgotten it in favor of linter dependence. Thanks for the reminder :)

I have some time so I'm gonna try to have this done today.